src/screens/Assets/wallet/WalletTab.js

```diff
@@ -122,13 +123,14 @@ function WalletTab() {
   };
 
   const hasPositiveBalance = totalBalance.value.gt(0);
+
   const buttons = [
     hasPositiveBalance && {
       title: tRoot('button.receive'),
       iconName: 'qrcode',
       onPress: showReceiveModal,
     },
-    hasPositiveBalance && {
+    isExchangeAvailable && hasPositiveBalance && {
```

Comment:
Just as per comment before – can we move inside-array logical operators outside? This results in arrays having falsy values and can easily slip somewhere unfiltered therefore in some scenarios causing crash.

Reply:
This is tightly scoped to `<FloatingButtons>` with proper type checking, I would keep it as is, because it's much simpler to define it this way then do `let array; if(xxx) array.push(yyy)`. This is established pattern with things likes React Native `style` prop which also accepts array with falsy values.